plugins/search-backend-module-catalog/README.md

```diff
@@ -20,22 +27,50 @@ backend.add(searchModuleCatalogCollator());
 backend.start();
 ```
 
-### Use custom schedule
+You may also want to add configuration parameters to your app-config, for example for controlling the scheduled indexing interval. These parameters should be placed under the `search.collators.catalog` key. See [the config definition file](https://github.com/backstage/backstage/blob/master/plugins/search-backend-module-catalog/config.d.ts) for more details.
+
+## Advanced Customizations
+
+This module also has an extension point, which lets you inject advanced customizations. Here's an example of how to leverage that extension point to tweak the transformer used for building the search indexer documents:
 
 ```tsx
 // packages/backend-next/src/index.ts
 import { createBackend } from '@backstage/backend-defaults';
+import { createBackendModule } from '@backstage/backend-plugin-api';
 import { searchPlugin } from '@backstage/plugin-search-backend/alpha';
-import { searchModuleCatalogCollator } from '@backstage/plugin-search-backend-module-catalog/alpha';
+import { CatalogCollatorEntityTransformer } from '@backstage/plugin-search-backend-module-catalog';
+import {
+  searchModuleCatalogCollator,
+  catalogCollatorExtensionPoint,
+} from '@backstage/plugin-search-backend-module-catalog/alpha';
 
-const schedule = {
-  frequency: { minutes: 10 },
-  timeout: { minutes: 15 },
-  initialDelay: { seconds: 3 },
-};
+const customTransformer: CatalogCollatorEntityTransformer = entity => ({
+  title: entity.metadata.title || entity.metadata.name,
+  text: entity.metadata.description || '',
+  componentType: entity.spec?.type?.toString() || 'other',
+  type: entity.spec?.type?.toString() || 'other',
+  namespace: entity.metadata.namespace || 'default',
+  kind: entity.kind,
+  lifecycle: (entity.spec?.lifecycle as string) || '',
+  owner: (entity.spec?.owner as string) || '',
+});
```

Comment:
Could be nicer to online this in `setEntityTransformer`?

Reply:
Had it there first, but the thing is that then it gets so mingled with the createBackendModule boilerplate. I felt it made things easier to read when it was two blocks with the separated concerns (definition vs injection).

Reply:
Essentially if the transformer were fewer lines of code with a smaller amount of symbol mess in it, then yeah, it'd have been down there